src/pb_stub.cc

```diff
@@ -934,6 +1040,18 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
           "send", &ResponseSender::Send, py::arg("response") = nullptr,
           py::arg("flags") = 0);
 
+  py::class_<Logger> logger(module, "Logger");
+  py::enum_<LogLevel>(logger, "LogLevel")
+      .value("INFO", LogLevel::INFO)
+      .value("WARNING", LogLevel::WARNING)
+      .value("ERROR", LogLevel::ERROR)
+      .value("VERBOSE", LogLevel::VERBOSE)
+      .export_values();
+  logger.def(py::init<>());
```

Comment:
Don't expose the constructor to the Python side. Instead, try to make the logger class in C++ a singleton and only expose the singleton instance to C++. The discussion here might be helpful: 
https://github.com/pybind/pybind11/issues/1418